clusterloader2/api/validation.go

```diff
@@ -243,5 +243,9 @@ func (v *ConfigValidator) validateGlobalQPSLoad(gl *GlobalQPSLoad, fldPath *fiel
 func (v *ConfigValidator) fileExists(path string) bool {
 	cwd, _ := os.Getwd()
 	_, err := os.Stat(fmt.Sprintf("%s/%s/%s", cwd, v.configDir, path))
+	if os.IsNotExist(err) {
```

Comment:
There's a function to check for abs path https://golang.org/pkg/path/filepath/#IsAbs

Reply:
avoiding additional import since we're already using os.Stat() elsewhere in this file